RevenueCatUI/CustomerCenter/ViewModels/PurchaseHistory/PurchaseHistoryViewModel.swift

```diff
@@ -121,6 +121,7 @@ private extension PurchaseHistoryViewModel {
                 transaction: subscription.value,
                 customerInfo: customerInfo,
                 purchasesProvider: purchasesProvider,
+                changePlans: [], // ignored on purpose because there's no change plan flow from history
```

Comment:
what does " from history" mean?

Reply:
I meant from `PurchaseHistory`, so we can ignore it in this screen. I'll make it more explicit